packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -168,42 +196,44 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
     return this.searchBoxState.suggestions.map((suggestion, index) => {
       const id = `${this._id}-suggestion-${index}`;
       return (
-        <button
-          type="button"
-          tabIndex={-1}
+        <li
+          role="option"
           onClick={(e) => this.onClickSuggestion(e)}
           onMouseDown={(e) => e.preventDefault()}
           part="suggestion"
           id={id}
-          value={suggestion.rawValue}
+          class="suggestion h-input px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none hover:bg-light-grey flex flex-row items-center"
           innerHTML={suggestion.highlightedValue}
-        ></button>
+          value={index}
```

Comment:
Hmm, why are we rendering the index here?

Reply:
The `onSelectValue` method needs a way to get the suggestion's rawValue, since the `HtmlLiElement` has value typed to a number, I couldn't pass in raw value to the value attribute of the `li` so I passed in the index so we could retrieve the raw value from the controller's state.